tests/sample_output.txt

```diff
@@ -14,6 +14,7 @@ Skipped lines:
   Feture: Login - Positive
   Bug: 123456
   Statues: Manual
+  Types: Functional
```

Comment:
Adding types to skipped lines output means that it is ensuring that Testimony will not parse type as types.